lib/livebook/hubs.ex

```diff
@@ -129,7 +129,7 @@ defmodule Livebook.Hubs do
 
   defp disconnect_hub(hub) do
     Task.Supervisor.start_child(Livebook.TaskSupervisor, fn ->
```

Comment:
```suggestion
    # We use a task supervisor because the hub connection itself
    # calls delete_hub (which calls this function), otherwise we deadlock.
    Task.Supervisor.start_child(Livebook.TaskSupervisor, fn ->
```